docs/node/task-queues.md

```diff
@@ -0,0 +1,56 @@
+---
+id: task-queues
+title: Task Queues in Node
+sidebar_label: Task Queues
+---
+
+## What is a Task Queue?
+
+import SharedTaskQueuesBasic from '../shared/task-queues-basic.md'
+
+<SharedTaskQueuesBasic
+workflowLink="/docs/java/workflows"
+workerLink="/docs/java/workers"
+/>
+
+## How to use Task Queues
+
+In Node, a Task Queue is represented in code by name, as a `string`.
+There are 3 places where the name of the Task Queue is supplied by the developer.
+
+1. When starting a Workflow, you must pass the `taskQueue` option to the [Connection's `workflow()` method](https://nodejs.temporal.io/api/classes/client.connection#workflow).
+
+```typescript
+const workflow = connection.workflow("my-workflow", {
+  taskQueue: "my-task-queue",
+});
+
+const result = await workflow.start();
+```
```

Comment:
This is now outdated with the recent changes to the client package.
Let's remember to update this doc before we release the new package.